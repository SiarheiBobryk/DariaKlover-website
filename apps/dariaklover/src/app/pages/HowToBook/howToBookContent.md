# –ö–∞–∫ –∑–∞–ø–∏—Å–∞—Ç—å—Å—è –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é

**–®–∞–≥ 1**. –ü–µ—Ä–µ–π–¥–∏—Ç–µ –Ω–∞ —Å—Ç—Ä–∞–Ω–∏—Ü—É [–ó–∞–ø–∏—Å–∞—Ç—å—Å—è](/booking).

**–®–∞–≥ 2**. –í—ã–±–µ—Ä–∏—Ç–µ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é –∏ –Ω–∞–∂–º–∏—Ç–µ –Ω–∞ –∫–Ω–æ–ø–∫—É _–í–´–ë–†–ê–¢–¨ –î–ê–¢–£ –ò –í–†–ï–ú–Ø_.

**–®–∞–≥ 3**. –í –æ—Ç–∫—Ä—ã–≤—à–µ–º—Å—è –æ–∫–Ω–µ —É–±–µ–¥–∏—Ç–µ—Å—å, —á—Ç–æ –æ—Ç–æ–±—Ä–∞–∂–∞–µ–º–æ–µ –≤—Ä–µ–º—è —Å–æ–æ—Ç–≤–µ—Ç—Å—Ç–≤—É–µ—Ç –≤–∞—à–µ–º—É –º–µ—Å—Ç–Ω–æ–º—É –≤—Ä–µ–º–µ–Ω–∏. –ï—Å–ª–∏ —ç—Ç–æ –Ω–µ —Ç–∞–∫, —Ç–æ —É—Å—Ç–∞–Ω–æ–≤–∏—Ç–µ –≤–∞—à —á–∞—Å–æ–≤–æ–π –ø–æ—è—Å.

![–£—Å—Ç–∞–Ω–æ–≤–∫–∞ —á–∞—Å–æ–≤–æ–≥–æ –ø–æ—è—Å–∞](how-to-book-1 '–£—Å—Ç–∞–Ω–æ–≤–∫–∞ –≤–∞—à–µ–≥–æ —á–∞—Å–æ–≤–æ–≥–æ –ø–æ—è—Å–∞')

**–®–∞–≥ 4**. –í—ã–±–µ—Ä–∏—Ç–µ –¥–µ–Ω—å.

![–í—ã–±–æ—Ä –¥–Ω—è](how-to-book-2 '–í—ã–±–æ—Ä –¥–Ω—è')

**–®–∞–≥ 5.** –í—ã–±–µ—Ä–∏—Ç–µ –≤—Ä–µ–º—è –∏ –ø–æ–¥—Ç–≤–µ—Ä–¥–∏—Ç–µ –≤—ã–±—Ä–∞–Ω–Ω–æ–µ –≤—Ä–µ–º—è –Ω–∞–∂–∞–≤ –Ω–∞ –∫–Ω–æ–ø–∫—É _Confirm_.

![–í—ã–±–æ—Ä –≤—Ä–µ–º–µ–Ω–∏](how-to-book-3 '–í—ã–±–æ—Ä –≤—Ä–µ–º–µ–Ω–∏')

![–ü–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ –≤—ã–±—Ä–∞–Ω–Ω–æ–≥–æ –≤—Ä–µ–º–µ–Ω–∏](how-to-book-4 '–ü–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ –≤—ã–±—Ä–∞–Ω–Ω–æ–≥–æ –≤—Ä–µ–º–µ–Ω–∏')

**–®–∞–≥ 6**. –í–≤–µ–¥–∏—Ç–µ —Å–≤–æ–µ –∏–º—è –≤ –ø–æ–ª–µ _Name_ [1], —ç–ª–µ–∫—Ç—Ä–æ–Ω–Ω—É—é –ø–æ—á—Ç—É –≤ –ø–æ–ª–µ _Email_ [2] –∏ –Ω–∞–∂–º–∏—Ç–µ –∫–Ω–æ–ø–∫—É _Schedule Event_ [3].

–ü–æ–∂–∞–ª—É–π—Å—Ç–∞, –≤–Ω–∏–º–∞—Ç–µ–ª—å–Ω–æ –≤–≤–æ–¥–∏—Ç–µ –∞–¥—Ä–µ—Å –≤–∞—à–µ–π —ç–ª–µ–∫—Ç—Ä–æ–Ω–Ω–æ–π –ø–æ—á—Ç—ã (_Email_), —Ç.–∫. –Ω–∞ —ç—Ç–æ—Ç –∞–¥—Ä–µ—Å –º—ã –≤–∞–º –≤—ã—à–ª–µ–º –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é —Å —É–∫–∞–∑–∞–Ω–∏–µ–º –≤—ã–±—Ä–∞–Ω–Ω–æ–≥–æ –¥–Ω—è –∏ –≤—Ä–µ–º–µ–Ω–∏.

![–í–≤–æ–¥ –≤–∞—à–µ–≥–æ –∏–º–µ–Ω–∏, —ç–ª–µ–∫—Ç—Ä–æ–Ω–Ω–æ–π –ø–æ—á—Ç—ã –∏ –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ](how-to-book-5 '–í–≤–æ–¥ –≤–∞—à–µ–≥–æ –∏–º–µ–Ω–∏, —ç–ª–µ–∫—Ç—Ä–æ–Ω–Ω–æ–π –ø–æ—á—Ç—ã –∏ –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ')

**–®–∞–≥ 7**. –í—ã —É–≤–∏–¥–∏—Ç–µ –ø–æ—Ö–æ–∂—É—é —Å—Ç—Ä–∞–Ω–∏—Ü—É, –≥–¥–µ –±—É–¥–µ—Ç –Ω–∞–¥–ø–∏—Å—å _Confirmed_ –∏ –ø–æ–¥—Ä–æ–±–Ω–∞—è –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è –æ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–∏.

![–ü–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ –∑–∞–ø–∏—Å–∏](how-to-book-6 '–ü–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ –∑–∞–ø–∏—Å–∏')

**–®–∞–≥ 8**. –ü—Ä–æ–≤–µ—Ä—å—Ç–µ –≤—Ö–æ–¥—è—â–∏–µ —Å–æ–æ–±—â–µ–Ω–∏—è –Ω–∞ –≤–∞—à–µ–π —ç–ª–µ–∫—Ç—Ä–æ–Ω–Ω—É—é –ø–æ—á—Ç–µ. –í–∞–º –¥–æ–ª–∂–Ω–æ –ø—Ä–∏–π—Ç–∏ —Å–æ–æ–±—â–µ–Ω–∏–µ —Å –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ–º –∑–∞–ø–∏—Å–∏ –≤ —Ç–µ—á–µ–Ω–∏–µ 5-10 –º–∏–Ω—É—Ç (–∏–Ω–æ–≥–¥–∞ —Å—Ä–∞–∑—É).

–ï—Å–ª–∏ –≤ –ø–∞–ø–∫–µ –í—Ö–æ–¥—è—â–∏–µ –Ω–∏—á–µ–≥–æ –Ω–µ—Ç, –ø–æ–ø—Ä–æ–±—É–π—Ç–µ –ø—Ä–æ–≤–µ—Ä–∏—Ç—å –ø–∞–ø–∫—É _–°–ø–∞–º_. –ò–Ω–æ–≥–¥–∞, —Å–æ–æ–±—â–µ–Ω–∏–µ –º–æ–∂–µ—Ç –±—ã—Ç—å —Ç–∞–º.

–í–∞–º –ø—Ä–∏–¥–µ—Ç –ø–æ—Ö–æ–∂–µ–µ —Å–æ–æ–±—â–µ–Ω–∏–µ —Å –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ–º –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é:

![–°–æ–æ–±—â–µ–Ω–∏–µ —Å –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ–º –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é](how-to-book-7 '–°–æ–æ–±—â–µ–Ω–∏–µ —Å –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ–º –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é')

–í –ø–æ–ª—É—á–µ–Ω–Ω–æ–º —Å–æ–æ–±—â–µ–Ω–∏–∏ –≤—ã —Å–º–æ–∂–µ—Ç–µ —É–≤–∏–¥–µ—Ç—å —Å–ª–µ–¥—É—é—â–µ–µ: –Ω–∞–∑–≤–∞–Ω–∏–µ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–∏ [1] (_Event Name_), –∫—Ä–∞—Ç–∫–æ–µ –æ–ø–∏—Å–∞–Ω–∏–µ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–∏ [2], –∫–Ω–æ–ø–∫—É –¥–ª—è –ø–æ–¥–∫–ª—é—á–µ–Ω–∏—è –∫ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–∏ [3] (_Join with Google Meet_), –¥–∞—Ç—É –∏ –≤—Ä–µ–º—è –ø—Ä–æ–≤–µ–¥–µ–Ω–∏—è –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–∏ [6], –∞ —Ç–∞–∫–∂–µ —Å—Å—ã–ª–∫—É –¥–ª—è –æ—Ç–º–µ–Ω—ã [4] (_Cancel_) –∏ –ø–µ—Ä–µ–Ω–æ—Å–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–∏ –Ω–∞ –¥—Ä—É–≥–æ–µ –≤—Ä–µ–º—è [5] (_Reschedule_).

![–í–∞–∂–Ω—ã–µ —ç–ª–µ–º–µ–Ω—Ç—ã –≤ —Å–æ–æ–±—â–µ–Ω–∏–∏ —Å –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ–º –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é](how-to-book-8 '–í–∞–∂–Ω—ã–µ —ç–ª–µ–º–µ–Ω—Ç—ã –≤ —Å–æ–æ–±—â–µ–Ω–∏–∏ —Å –ø–æ–¥—Ç–≤–µ—Ä–∂–¥–µ–Ω–∏–µ–º –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é')

–ü–æ–∑–¥—Ä–∞–≤–ª—è–µ–º, —É –≤–∞—Å –ø–æ–ª—É—á–∏–ª–æ—Å—å üéâ

–°–æ—Ö—Ä–∞–Ω–∏—Ç–µ —Å–µ–±–µ –≤ –∫–∞–ª–µ–Ω–¥–∞—Ä–µ –¥–∞—Ç—É –∏ –≤—Ä–µ–º—è –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–∏.

–î–æ –≤—Å—Ç—Ä–µ—á–∏!
