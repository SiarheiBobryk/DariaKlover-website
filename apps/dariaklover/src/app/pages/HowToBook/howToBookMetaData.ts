export interface HowToBookMetaData {
  title: string;
  heading: string;
  path: string;
}

const howToBookMetaData: HowToBookMetaData = {
  title: 'Как записаться на консультацию',
  heading: 'Как записаться на консультацию',
  path: '/help/how-to-book',
};

export default howToBookMetaData;
