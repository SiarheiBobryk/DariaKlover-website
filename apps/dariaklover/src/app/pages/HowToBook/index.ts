export { default } from './HowToBook';

export { default as howToBookMetaData } from './howToBookMetaData';
export * from './howToBookMetaData';
