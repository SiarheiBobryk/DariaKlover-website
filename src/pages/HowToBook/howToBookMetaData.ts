export interface HowToBookMetaData {
  title: string;
  heading: string;
  path: string;
}

const howToBookMetaData: HowToBookMetaData = {
  title: 'Как записаться на консультацию',
  heading: 'Как записаться на консультацию',
  path: '/how-to-book',
};

export default howToBookMetaData;
