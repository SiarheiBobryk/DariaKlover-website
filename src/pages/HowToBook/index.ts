export { default } from './HowToBook';
export { default as howToBookMetaData } from './howToBookMetaData';
