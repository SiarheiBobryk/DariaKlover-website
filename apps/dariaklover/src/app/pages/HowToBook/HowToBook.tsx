import * as React from 'react';
import { Helmet } from 'react-helmet';

import Alert from '@mui/material/Alert';
import AlertTitle from '@mui/material/AlertTitle';

import AlertHelp from '../../components/AlertHelp';
import howToBookMetaData from './howToBookMetaData';
import Markdown from '../../components/Markdown';
import howToBookContent from './howToBookContent.md';

function HowToBook() {
  return (
    <>
      <Helmet>
        <title>{howToBookMetaData.title}</title>
      </Helmet>
      <Markdown>{howToBookContent}</Markdown>
      {/* SOMEDAY: Move these "Alert"s to the MD content. Proper mapping is required */}
      <Alert severity="info" variant="outlined">
        <AlertTitle>Русский язык</AlertTitle>
        Система записи на консультацию находится на стороннем сервисе, который очень простой и понятный, однако этот
        сервис не имеет русского языка. Для тех, кто не говорит по-английски, мы очень постарались перевести на русский
        язык все элементы на страничках, но, к сожалению, на данный момент технически не возможно предоставить 100%
        русскую версию.
      </Alert>
      <AlertHelp />
    </>
  );
}

export default HowToBook;
