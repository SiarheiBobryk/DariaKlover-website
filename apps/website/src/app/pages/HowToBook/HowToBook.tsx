import * as React from 'react';
import { Helmet } from 'react-helmet';

import HowToBookMDX from './howToBook.mdx';
import howToBookMetaData from './howToBookMetaData';

function HowToBook() {
  return (
    <>
      <Helmet>
        <title>{howToBookMetaData.title}</title>
      </Helmet>
      <HowToBookMDX />
    </>
  );
}

export default HowToBook;
