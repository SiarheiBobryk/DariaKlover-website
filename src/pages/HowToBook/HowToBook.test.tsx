import * as React from 'react';
import { BrowserRouter } from 'react-router-dom';

import { axe, toHaveNoViolations } from 'jest-axe';
import { render, RenderResult } from '@testing-library/react';
import renderer, { ReactTestRendererJSON } from 'react-test-renderer';

import HowToBook from './HowToBook';

expect.extend(toHaveNoViolations);

describe('<HowToBook />', () => {
  describe('a11y', () => {
    it('should match a11y standards', async () => {
      const { container }: RenderResult = render(
        <BrowserRouter>
          <HowToBook />
        </BrowserRouter>,
      );

      const results = await axe(container);
      expect(results).toHaveNoViolations();
    });
  });

  describe('snapshotting', () => {
    it('should render correctly', () => {
      const howToBook: ReactTestRendererJSON | ReactTestRendererJSON[] | null = renderer
        .create(
          <BrowserRouter>
            <HowToBook />
          </BrowserRouter>,
        )
        .toJSON();
      expect(howToBook).toMatchSnapshot();
    });
  });
});
