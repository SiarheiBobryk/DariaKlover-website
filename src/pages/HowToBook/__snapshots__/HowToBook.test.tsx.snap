// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HowToBook /> snapshotting should render correctly 1`] = `
Array [
  <h2
    className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
  >
    Как записаться на консультацию
  </h2>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 1
    </strong>
    . Перейдите на страницу
     
    <a
      className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
      href="/booking"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
    >
      Запись на консультацию
    </a>
    .
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 2
    </strong>
    . Выберите консультацию и нажмите на кнопку 
    <i>
      ВЫБРАТЬ ДАТУ И ВРЕМЯ
    </i>
    .
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 3
    </strong>
    . В открывшемся окне убедитесь, что отображаемое время соответствует вашему местному времени. Если это не так, то установите ваш часовой пояс.
  </p>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-1.webp"
        type="image/webp"
      />
      <img
        alt="Установка часового пояса"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-1.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Установка вашего часового пояса
      </p>
    </figcaption>
  </figure>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 4
    </strong>
    . Выберите день.
  </p>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-2.webp"
        type="image/webp"
      />
      <img
        alt="Выбор дня"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-2.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Выбор дня
      </p>
    </figcaption>
  </figure>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 5
    </strong>
    . Выберите время и подтвердите выбранное время нажав на кнопку 
    <i>
      Confirm
    </i>
    .
  </p>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-3.webp"
        type="image/webp"
      />
      <img
        alt="Выбор времени"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-3.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Выбор времени
      </p>
    </figcaption>
  </figure>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-4.webp"
        type="image/webp"
      />
      <img
        alt="Подтверждение выбранного времени"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-4.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Подтверждение выбранного времени
      </p>
    </figcaption>
  </figure>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 6
    </strong>
    . Введите свое имя в поле 
    <i>
      Name
    </i>
     [1], электронную почту в поле 
    <i>
      Email
    </i>
     [2] и нажмите кнопку 
    <i>
      Schedule Event
    </i>
     [3].
  </p>,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardWarning MuiAlert-standard css-rob5aa-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="ReportProblemOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 5.99L19.53 19H4.47L12 5.99M12 2L1 21h22L12 2zm1 14h-2v2h2v-2zm0-6h-2v4h2v-4z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message css-1pxa9xg-MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        Электронная почта (
        <i>
          Email
        </i>
        )
      </div>
      Пожалуйста, внимательно вводите адрес вашей электронной почты (
      <i>
        Email
      </i>
      ), т.к. на этот адрес мы вам вышлем подтверждение записи на консультацию с указанием выбранного дня и времени.
    </div>
  </div>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-5.webp"
        type="image/webp"
      />
      <img
        alt="Ввод вашего имени, электронной почты и подтверждение"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-5.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Ввод вашего имени, электронной почты и подтверждение
      </p>
    </figcaption>
  </figure>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 7
    </strong>
    . Вы увидите похожую страницу, где будет надпись 
    <i>
      Confirmed
    </i>
     и подробная информация о консультации.
  </p>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-6.webp"
        type="image/webp"
      />
      <img
        alt="Подтверждение записи"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-6.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Подтверждение записи
      </p>
    </figcaption>
  </figure>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    <strong>
      Шаг 8
    </strong>
    . Проверьте входящие сообщения на вашей электронную почте. Вам должно прийти сообщение с подтверждением записи в течение 5-10 минут (иногда сразу).
  </p>,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardWarning MuiAlert-standard css-rob5aa-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="ReportProblemOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 5.99L19.53 19H4.47L12 5.99M12 2L1 21h22L12 2zm1 14h-2v2h2v-2zm0-6h-2v4h2v-4z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message css-1pxa9xg-MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        Папка 
        <i>
          Спам
        </i>
      </div>
      Если в папке 
      <i>
        Входящие
      </i>
       ничего нет, попробуйте проверить папку 
      <i>
        Спам
      </i>
      . Иногда, сообщение может быть там.
    </div>
  </div>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    Вам придет похожее сообщение с подтверждением записи на консультацию:
  </p>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-7.webp"
        type="image/webp"
      />
      <img
        alt="Сообщение с подтверждением записи на консультацию"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-7.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Сообщение с подтверждением записи на консультацию
      </p>
    </figcaption>
  </figure>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    В полученном сообщении вы сможете увидеть следующее: название консультации [1] (
    <i>
      Event Name
    </i>
    ), краткое описание консультации [2], кнопку для подключения к консультации [3] (
    <i>
      Join with Google Meet
    </i>
    ), дату и время проведения консультации [6], а также ссылку для отмены [4] (
    <i>
      Cancel
    </i>
    ) и переноса консультации на другое время [5] (
    <i>
      Reschedule
    </i>
    ).
  </p>,
  <figure
    className="css-b9qyun"
  >
    <picture>
      <source
        srcSet="/responsive_images/how-to-book-8.webp"
        type="image/webp"
      />
      <img
        alt="Важные элементы в сообщении с подтверждением записи на консультацию"
        className="css-gkqd5w"
        loading="lazy"
        src="/responsive_images/how-to-book-8.png"
      />
    </picture>
    <figcaption>
      <p
        className="MuiTypography-root MuiTypography-caption css-1v2k9m3-MuiTypography-root"
      >
        Важные элементы в сообщении с подтверждением записи на консультацию
      </p>
    </figcaption>
  </figure>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    Поздравляем, у вас получилось 🎉
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    Сохраните себе в календаре дату и время консультации.
  </p>,
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
  >
    До встречи!
  </p>,
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardInfo MuiAlert-standard css-12mvehg-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="InfoOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message css-1pxa9xg-MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        Связаться с нами
      </div>
      Если у вас остались вопросы или вы продолжаете испытывать сложности в работе с сайтом, то не стесняйтесь и свяжитесь с нами через
       
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
        href="https://t.me/dariaklover"
        onBlur={[Function]}
        onFocus={[Function]}
        rel="noopener"
        target="_blank"
      >
        Telegram
         
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-1xzgpfl-MuiSvgIcon-root"
          data-testid="OpenInNewIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
          />
        </svg>
      </a>
      ,
       
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
        href="https://wa.me/+375447841051"
        onBlur={[Function]}
        onFocus={[Function]}
        rel="noopener"
        target="_blank"
      >
        WhatsApp
         
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-1xzgpfl-MuiSvgIcon-root"
          data-testid="OpenInNewIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
          />
        </svg>
      </a>
       
      или по
       
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
        href="mailto: dariaklover@gmail.com"
        onBlur={[Function]}
        onFocus={[Function]}
        rel="noopener"
        target="_blank"
      >
        электронной почте (
        <i>
          Email
        </i>
        )
         
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-1xzgpfl-MuiSvgIcon-root"
          data-testid="OpenInNewIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
          />
        </svg>
      </a>
      . Мы обязательно поможем.
    </div>
  </div>,
]
`;
